app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -214,6 +225,12 @@ public Cursor getAlbums(String artist) {
         return buildSelectSql(LVL_ALBUM);
     }
 
+    public Cursor getAlbumsOrderedByYear(String artist) {
```

Comment:
Make a `getAlbum` which as a `LibraryAlbumOrder` parameter as well as the artist.